target/snitch_cluster/README.md

```diff
@@ -45,18 +48,26 @@ make CFG_OVERRIDE=cfg/custom.hjson bin/snitch_cluster.vlt
 
 ___Note:__ whenever you override the configuration file on the `make` command-line, the configuration will be stored in the `cfg/lru.hjson` file. Successive invocations of `make` will automatically pick up the `cfg/lru.hjson` file. You can therefore omit the `CFG_OVERRIDE` definition in successive commands unless you want to override the least-recently used configuration._
 
+Banshee uses also a cluster configuration file, however, that is given directly when simulating a specific binary with banshee with the help of the `--configuration <cluster_config.yaml>`.
+
 ### Building the software
 
 To build all of the software for the Snitch cluster, run the following command:
 
 ```bash
+# for RTL simulation
 make DEBUG=ON sw
+
+# for Banshee simulation (requires slightly different runtime)
+make SELECT_RUNTIME=banshee DEBUG=ON sw
 ```
 
 The `sw` target first generates some C header files which depend on the hardware configuration. Hence, the need to generate the software for the same configuration as your hardware. Afterwards, it recursively invokes the `make` target in the `sw` subdirectory to build the apps/kernels which have been developed in that directory.
 
 The `DEBUG=ON` flag is used to tell the compiler to produce debugging symbols. It is necessary for the `annotate` target, showcased in the Debugging section of this guide, to work.
 
+The `SELECT_RUNTIME` is set by default to `rtl`. To use the Banshee runtime, it requires to be set to be set to `banshee` with `SELECT_RUNTIME=banshee`.
```

Comment:
This sentence repeats "to be set". Also, it's a bit hard to follow.

Reply:
"The `SELECT_RUNTIME` flag" or just "`SELECT_RUNTIME`"